dockerfile

```diff
@@ -18,12 +21,12 @@ FROM base as production
 
 EXPOSE 5000
 
-ENTRYPOINT [ "poetry","run","gunicorn","todo_app.app:create_app()" ,"--bind","0.0.0.0"]
+ENTRYPOINT ["./entrypoint.sh"]
 
 #local development stage
 FROM base as development
 
-EXPOSE 5000
+EXPOSE 8000
```

Comment:
I'm not sure why we switched this - this should be referring to the development container which still uses 5000 as the default, and actually it looks like your production container _used_ to use 8000 but now uses 5000 so I think the one above was incorrect before but now right 😄